.gitignore

```diff
@@ -7,6 +7,7 @@ terraform/aws/terraform.tfvars
 terraform/azure/.terraform
 terraform/azure/terraform.tfstate*
 terraform/azure/terraform.tfvars
+terraform/azure/fence_data.json
```

Comment:
When using the azure fence agent, we need additional information when creating the cluster.  This information is known to terraform but not Ansible.  Therefore the file ./terraform/azure/fence_data.json is created.  When using azure native fencing, this file can be sourced to get the sub id, tenant id and resource group.  It's added to .gitignore as it contains sensitive data.